aptos-move/aptos-vm-types/src/module_write_set.rs

```diff
@@ -57,12 +60,24 @@ impl ModuleWriteSet {
     pub fn write_op_info_iter_mut<'a>(
         &'a mut self,
         executor_view: &'a dyn ExecutorView,
+        module_storage: &'a impl AptosModuleStorage,
     ) -> impl Iterator<Item = PartialVMResult<WriteOpInfo>> {
-        self.write_ops.iter_mut().map(|(key, op)| {
+        self.write_ops.iter_mut().map(move |(key, op)| {
+            let prev_size = if module_storage
```

Comment:
I guess this is the compatibility issue you mentioned previously?

Reply:
No, this is just a V2 path which is separate from data (so we resolve directly through module storage, not through ExecutorView trait.